traffic_portal/app/src/common/api/AuthService.js

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-var AuthService = function($rootScope, $http, $state, $location, $q, $state, httpService, userModel, messageModel, ENV) {
+var AuthService = function($rootScope, $http, $state, $location, $q, httpService, userModel, messageModel, ENV) {
```

Comment:
good catch, however, this change probably won't compile or will cause strange errors because you didn't update line 79 as well.